Source/ScrollViewAdapters/EmptyContentScrollViewAdapter.swift

```diff
@@ -166,4 +166,28 @@ open class EmptyContentScrollViewAdapter<State: Equatable, View: UIScrollView>:
         self.button()?.action()
     }
     
+    public func emptyDataSetShouldAllowTouch(_ scrollView: UIScrollView!) -> Bool {
+        guard let shouldAllowTouch = emptyContentDataSource?.emptyContent(for: stateManaging.state)?.shouldAllowTouch else {
```

Comment:
I think it is a good idea to implement the delegate here.

Instead of using the content I would consider the following approaches:
1. Instead of variables on the content, create them on `EmptyContnetntScrollViewAdapter`. By doing so, we loose the possibility to enable/disable scrolling per content.
2. Create a Delegate which can be asked if we should scroll or not (`EmptyContnetntScrollViewAdapterDelegate`)

Reply:
I would not prefer the first approach, as you mentioned we will loose the possibility to enable/disable scrolling per content.

Which class should class should hold a reference to the delegate implementation? EmptyContent?

Reply:
`EmptyContnetntScrollViewAdapter` holds a reference to its delegate. This delegate can be asked for scrolling/touching/etc